test/ecto/adapters/mysql_test.exs

```diff
@@ -599,11 +600,11 @@ defmodule Ecto.Adapters.MySQLTest do
   test "create table with references" do
     create = {:create, table(:posts),
                [{:add, :id, :serial, [primary_key: true]},
-                {:add, :category_0, references(:categories), []},
-                {:add, :category_1, references(:categories, name: :foo_bar), []},
-                {:add, :category_2, references(:categories, on_delete: :nothing), []},
-                {:add, :category_3, references(:categories, on_delete: :delete_all), [null: false]},
-                {:add, :category_4, references(:categories, on_delete: :nilify_all), []}]}
+                {:add, :category_0, struct(%Reference{table: :categories}, []), []},
```

Comment:
You don't need to use `struct(%Reference{table: :categories}, name: :foo_bar)`, instead you can write it as `%Reference{table: :categories, name: :foo_bar}` and it will be a bit more concise. :)